centos_7_py2py3/cloudify-wagon-builder.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+echo "Starting..."
+
+CONSTRAINTS_FILE=/packaging/constraints.txt
+REQUIRMENTS_FILE=/packaging/dev-requirements.txt
+
+if test -f /packaging/extra-packaging-instructions.sh
+then
+    source /packaging/extra-packaging-instructions.sh
+fi
+source /$1/bin/activate
+echo "manylinux1_compatible = False" > "/$1/bin/_manylinux.py"
+if test -f ${CONSTRAINTS_FILE}
```

Comment:
Best rewritten similar to the above if statement

Reply:
(but in this case don't forget to surround ${CONSTRAINTS_FILE} with "", in case it is empty.